ddtrace/ddtrace.stub.php

```diff
@@ -0,0 +1,1051 @@
+     * tracing information will be applied if available.
+     *
+     * @param float $startTime Start time of the span in seconds.
+     * @return SpanData|false The newly started span, or 'false' if a wrong parameter was given.
+     */
+    function start_span(float $startTime = 0): SpanData|false {}
+
+    /**
+     * Close the currently active user-span on the top of the stack
+     *
+     * @param float $finishTime Finish time in seconds. Defaults to now if zero.
+     * @return false|null 'false' if unexpected parameters were given, else 'null'
+     */
+    function close_span(float $finishTime = 0): false|null {}
+
+    /**
+     * Update the duration of an already closed span
+     *
+     * Note that this API won't cause an update of a closed span if it's already sent. Its usage, particularly on
+     * root spans with datadog.trace.auto_flush_enabled may not yield the expected results.
+     *
+     * @param SpanData $span The span to update.
+     * @param float $finishTime Finish time in seconds. Defaults to now if zero.
+     * @return false|null 'false' if unexpected parameters were given, else 'null'
+     */
+    function update_span_duration(SpanData $span, float $finishTime = 0): false|null {}
+
+    /**
+     * Start a new trace
+     *
+     * More precisely, a new root span stack will be created and switched on to, and a new span started.
+     *
+     * @param float $startTime Start time of the span in seconds.
+     * @return SpanData The newly created root span
+     */
+    function start_trace_span(float $startTime = 0): SpanData {}
+
+    /**
+     * Get the active stack
+     *
+     * @return SpanStack|null A copy of the active stack, or 'null' if the tracer is disabled. Won't happen
+     * under normal operation.
+     */
+    function active_stack(): SpanStack|null {}
+
+    /**
+     * Initialize a new span stack and switch to it. If tracing isn't enabled, a root span stack will be created.
+     *
+     * @return SpanStack The newly created span stack
+     */
+    function create_stack(): SpanStack {}
+
+    /**
+     * Switch back to a specific stack (even if there is no active span on that stack), or to the parent of the active
+     * stack if no stack is given.
+     *
+     * @param SpanData|SpanStack|null $newStack Stack to switch to. If 'null' is given, switches to the parent of the
+     * active stack. If a SpanData object is given, it will switch to the stack of the latter.
+     * @return null|false|SpanStack The newly active stack, or 'null' if the tracer is disabled (Won't happen under
+     * normal operation), or 'false' if unexpected parameters were given.
+     */
+    function switch_stack(SpanData|SpanStack|null $newStack = null): null|false|SpanStack {}
+
+    /**
+     * Set the priority sampling level
+     *
+     * @param int $priority The priority level to be set to.
+     * @param bool|null $global If set to 'true' and if there is no active stack (or the active stack doesn't have a
```

Comment:
Type of parameter `bool|null` differs from the one, declared in signature. Could you please update typehints?